warehouse/scripts/json_to_docblocks.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
so to clarify/confirm, this is what you used to automatically generate docblocks for the GTFS schedule data?

Reply:
yes, I will comment